autogen/docs/behaviorspace.md.mustache

```diff
@@ -227,25 +227,25 @@ the setting of **Measure runs at every step** option. In either case, the
-imported into a spreadsheet application.
-
-(Note however that spreadsheet data is not written to the results file until the
-experiment finishes. Since spreadsheet data is stored in memory until the
-experiment is done, very large experiments could run out of memory. So you
-should disable spreadsheet output unless you really want it. If you do want
-spreadsheet output, note that if anything interrupts the experiment, such as a
-runtime error, running out of memory, or a crash or power outage, no spreadsheet
-results will be written. For long experiments, you may want to also enable table
-format as a precaution so that if something happens and you get no spreadsheet
-output you'll at least get partial table output.)
+Both **Table output** and **Spreadsheet output** will include a header section that has rows for 1) the NetLogo app version used, 2) the name of the NetLogo model file used, 3) the name of the BehaviorSpace experiment used, 4) the date and time at the start, and 5) the dimensions of the world used at the start.
+
+The **Table output** format lists each measurement step from each run in its own row, with each metric in a separate column.  The measurement rows will appear in the order they happen in real time.  With the **parallel runs** option the measurements may appear in a mixed order as multiple runs can happen simultaneously.  To help identify which run a row belongs to, there is a column titled `[run number]` along with a column titled `[step]` that indicates the measurement number for that run.  Each row also includes the values used for the variables during initial setup of the run; these values are the same for each of that run's measurement rows.  **Table output** data is written to the file as each run completes.
+
+The **Table output** format is good to use when you want to further process the data using another tool, such as importing into a database, a statistics package, or a spreadsheet application for analysis.
+
+In the below **Table output** sample image the header section is in red, the run number and step columns are in blue, the initial values of the variables for the run are in green, and the measurement metric data is in purple.
+
+![[bs-table.png]]
+
+The **Spreadsheet output** format lists each metric for each run in a separate column, then each row corresponds to a measurement step that applies to all runs.  If one run finishes before another due to a **stop condition**, then its measurement steps after that point will be blank.  At the top of the file there is a `[run number]` row that will have the run number repeated for each metric at the top of the file.  After that are rows for the initial values given to each variable for the run.  The spreadsheet output also calculates the min, mean, max, and final values for each metric and lists those after the initial values before the individual run measurements.  Then it lists the actual number of steps a run went through in a `[steps]` row.
```

Comment:
I think "with each row corresponding to a measurement step"
would read better than "then each row corresponds to a measurement step"